stock_prices/stock_prices.py

```diff
@@ -3,7 +3,14 @@
 import argparse
 
 def find_max_profit(prices):
-  pass
+  minprice=prices[0]
+  profit=prices[1]-prices[0]
+  for i in range(1,len(prices)):
+    if prices[i]<minprice:
+      minprice=prices[i]
+    elif prices[i]-minprice>profit:
+      profit=prices[i]-minprice
+  return profit
 
```

Comment:
Really well done on an `O(n)` solution. The only thing is that this doesn't seem to be work in some cases, particularly if all the values of the list are descending. You are passing the second assertion, but if the test is modified like so and added to the testing file:
```
self.assertEqual(find_max_profit([100, 85, 80, 50, 20, 10]), -5)
```
Your implementation would return `-15`, whereas the expected is `-5` (buy at `85`, sell at `80` for the lowest possible loss/greatest possible profit).

By the way, this is not priority. Poke around with this once you have solutions for the other problems in this repo.